app/__init__.py

```diff
@@ -69,7 +73,7 @@ def add_generic_cors_header(response):
 @app.after_request
 def add_cache_control_header(response):
     # For /checker route we let the frontend proxy decide how to cache it.
-    if request.method == 'GET' and request.path != url_for('checker'):
+    if request.method == 'GET' and request.endpoint != 'checker':
```

Comment:
Out of curiosity: what's the reason for using `request.endpoint` instead of `url_for('checker')` here (and also in `add_generic_cors_header()`?

Reply:
Efficiency, to avoid a function call which is not necessary. `request.endpoint` is a string that already contains the name of the route function, so there is no need to use the function `url_for()`. A simple string compare is more efficient/performant as a function call associated with string compare.

Reply:
Makes sense, thanks for the explanation